src/main/java/org/testcontainers/containers/TarantoolContainer.java

```diff
@@ -17,9 +17,9 @@
 public class TarantoolContainer extends GenericContainer<TarantoolContainer>
         implements TarantoolContainerOperations<TarantoolContainer> {
 
-    public static final String TARANTOOL_IMAGE = "tarantool/tarantool";
+    public static final String DEFAULT_IMAGE_NAME = "tarantool/tarantool";
     public static final String DEFAULT_IMAGE_VERSION = "2.11.2-ubuntu20.04";
```

Comment:
```suggestion
    public static final String DEFAULT_TAG = "2.11.2-ubuntu20.04";
```

Если следовать именованию в https://docs.docker.com/reference/dockerfile/#from